ci-operator/step-registry/acm/interop-p2p/submariner-diag/acm-interop-p2p-submariner-diag-commands.sh

```
@@ -0,0 +1,211 @@
+    typeset kc="${1:?}"; (($#)) && shift
+    typeset name="${1:?}"; (($#)) && shift
+
+    typeset gatherWorkDir
+    gatherWorkDir="$(mktemp -d /tmp/subctl-gather-XXXXXX)"
+
+    (
+        cd "${gatherWorkDir}"
+        KUBECONFIG="${kc}" "${subctlBin}" gather > "${diagDir}/gather-${name}.log" 2>&1 || true
+
+        # Archive whatever subctl gather produced
+        typeset tarball=""
```

Shellcheck (0.11.0)

[warning] tarball appears unused. Verify use (or export if used externally).

(SC2034)